ApolloSQLite/SQLiteNormalizedCache.swift

```diff
@@ -83,60 +83,56 @@ public final class SQLiteNormalizedCache: NormalizedCache {
       throw SQLiteNormalizedCacheError.invalidRecordEncoding(record: record)
     }
 
-    let recordJSON = try SQLiteSerialization.deserialize(data: recordData)
-    return Record(key: row[key], recordJSON)
+    let fields = try SQLiteSerialization.deserialize(data: recordData)
+    return Record(key: row[key], fields)
   }
 }
 
 private let serializedReferenceKey = "reference"
 
 final class SQLiteSerialization {
-  static func serialize(fields: JSONObject) throws -> Data {
+  static func serialize(fields: Record.Fields) throws -> Data {
     var objectToSerialize = JSONObject()
     for (key, value) in fields {
       objectToSerialize[key] = try serialize(value: value)
     }
-    return try JSONSerializationFormat.serialize(value: objectToSerialize)
```

Comment:
The cleanup in this commit is great! Perhaps `JSONSerializationFormat` should have a more descriptive name, to make its purpose more distinct than the system-provided `JSONSerialization`?